src/main/StandaloneFilterWindow.h

```diff
@@ -601,6 +601,30 @@ class StandaloneFilterWindow    : public DocumentWindow
                                                         preferredDefaultDeviceName, preferredSetupOptions,
                                                         constrainToConfiguration, autoOpenMidiDevices));
 
+        class MyChangeListener : public juce::ChangeListener
```

Comment:
probably we can use a more distinctive name for this class

Reply:
Resolved in 85f62fddf647243bf7d74eee6e249c28b30371e0